distribution/tools/entitlements-agent/src/main/java/org/elasticsearch/entitlements/agent/Agent.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.entitlements.agent;
+
+import org.elasticsearch.entitlements.runtime.api.EntitlementChecks;
+
+import java.lang.instrument.Instrumentation;
+
+public class Agent {
```

Comment:
Should we name this `EntitlementAgent` or something a bit more descriptive to avoid overloaded terms?

Reply:
I don't really have an opinion here. Whatever you guys think.

Reply:
++ for EntitlementAgent - if anything, it make find in Intellij easier :)